### PR TITLE
Updated hotel rules

### LINKED ISSUE #155: Hotel Rules
from trello: https://trello.com/c/Kag6VS2V/20-minor-bugs
 - breakfast as well as phone number nor address are not a rule
 - checkin and checkout times should be in better format (e.g. "11:00")
 - "These are th hotel rules:" should be translated to English (probably something like "By booking you agree to the following terms and conditions.")
 - What is the purpose of the dialog, shouldn't we show checkbox under "Proceed with booking"? with the link to the dialog?

What about this section?
@skkr we shouldn't show the room's data here, I mean if the breakfast depends on the room type, we shouldn't show it as hotel rules.

So.. what about the hotel information we should show here?

```javascript
{
  "id":"0x4Ee2BC4804D1A75AEBa5C76D25cc2c036B136140",
  "name":"Hotel i31","
  description":"**Boutique Hotel i31 Berlin** – The 4 star superior hotel located in the heart of Berlin welcomes its guests with a modern design harmonised with Berlin’s relaxed lifestyle. The 121 hotel rooms are equipped with an individually adjustable air conditioning and offers free wifi (up to three devices), a minibar with free soft drinks,  including flat-screen smart TV and a media hub as well as an electronic safe. We offer our guests elegant, contemporary style rooms with individual design and high-quality interior.\nOur concept harmonises space, technology, design and comfort into a modern, stylish hotel in which leisure and business travellers can enjoy all the amenities just alike. ",
  "location":{
    "latitude":52.531015,
    "longitude":13.384402
  },
  "contacts":{
    "general":{
      "email":"info@hotel-i31.de",
      "phone":"0049303384000",
      "url":"https://www.hotel-i31.de/",
      "additionalContacts":[{"title":"twitter","value":"@Hoteli31"}]
    }
  },
  "address":{
    "line1":"Boutique Hotel i31",
    "line2":"Invalidenstraße31",
    "postalCode":"10115",
    "city":"Berlin",
    "country":"DE"
    },
  "currency":"EUR",
  "images":["https://www.hotel-i31.de/wp-content/uploads/sites/71/2017/06/i31-hotel-berlin-mitte-bar-e1505292828127.jpg","https://www.hotel-i31.de/wp-content/uploads/sites/71/2018/03/i31-hotel-berlin-mitte-garten-3.jpg","https://www.hotel-i31.de/wp-content/uploads/sites/71/2018/03/i31-hotel-berlin-mitte-gym-1.jpg"],
  "amenities":["Hofgarten","Bar","Fitness"],
  "updatedAt":"2018-07-24T10:15:00"
}
```

### PR DESCRIPTION
**Pull request by skkr**
closes #155

### CODE REVIEW

File: app/src/components/Booking/FormSection/RulesModal.jsx [hunk 1]
```diff
@@ -16,22 +16,16 @@ const RulesModal = () => (
...
             <li>
-              <b>Check in time</b>: 12-00
+              <b>Check in time</b>: 10am
             </li>
             <li>
-              <b>Check out time</b>: 11-00
+              <b>Check out time</b>: 11am
             </li>
             <li>
               <b>Breakfast</b>: included
```
Comment (PabloAlice) [inline app/src/components/Booking/FormSection/RulesModal.jsx]: @AugustoL breakfast lncluded is right?

Comment (AugustoL) [reply app/src/components/Booking/FormSection/RulesModal.jsx]: nope,

**Comment by skkr** [comment]
@AugustoL , @PabloAlice  done!

**Comment by AugustoL** [comment]
Fix #155

**Pull request merged**